src/app/server/Mdns.cpp

```diff
@@ -146,12 +148,16 @@ CHIP_ERROR AdvertiseCommisioning()
 
     auto & mdnsAdvertiser = chip::Mdns::ServiceAdvertiser::Instance();
 
+    ChipLogProgress(Discovery, "Advertise commission parameter vendorID=%u productID=%u discriminator=%04u/%02u",
```

Comment:
How about "Advertise commissionable node" to make the message more similar to the previous one?